### PR TITLE
Update instructions for Debian Buster

### LINKED ISSUE #349: bump up Debian stable instructions
Buster is the new stable version of Debian. [Buster's latest version of Cockpit is 188](https://www.debian.org/releases/buster/), which is [a year and a half old now](https://cockpit-project.org/blog/cockpit-188.html). People should be using a more recent version of Cockpit, so we should drop Stretch and switch it to Buster on the website.

### PR DESCRIPTION
**Pull request by garrett**
Fixes #349.

### CODE REVIEW

**Comment by garrett** [comment]
I had to rework alerts to allow code blocks inside (using backticks in Markdown, for example).

Preview should be at https://garrett.github.io/cockpit-project.github.io/running.html#debian soon.

**Review by martinpitt — APPROVED** [review]
Thanks! Good with the little version number fix.

File: running.md [hunk 1]
```diff
@@ -202,20 +202,22 @@ sudo firewall-cmd --reload
...
 ```
 
 2. Install the package:
 ```
-sudo apt-get install cockpit
+sudo apt install -t buster-backports cockpit
 ```
 
+{:.note}
+When installing Cockpit-related packages and any dependencies, make sure to use `-t buster-backports` so backports are included.
```
Comment (martinpitt) [inline running.md:220]: Hmm, I don't actually see that note on https://garrett.github.io/cockpit-project.github.io/running.html#debian ?

File: running.md [hunk 2]
```diff
@@ -202,20 +202,22 @@ sudo firewall-cmd --reload
 
 ### Debian
 
-Cockpit is included in Debian unstable and in backports for 9 (Stretch).
+Cockpit is included in Debian unstable and in backports for 9 (Buster).
```
[the remark this answers is not in the document]
Comment (garrett) [reply running.md:205]: Oops. Right.

File: running.md (hunk 1 above)
Comment (garrett) [reply running.md:220]: the site hasn't built. I don't know why. Weird.

Comment (garrett) [reply running.md:220]: :facepalm: I had an -n for previewing the force-push.

**Comment by garrett** [comment]
I bumped up 9 to 10, did a commit amend and a force push to this branch and master.

**Comment by garrett** [comment]
Force-push with fixes #, closes #.

File: running.md [hunk 3]
```diff
@@ -202,20 +202,22 @@ sudo firewall-cmd --reload
 
 ### Debian
 
-Cockpit is included in Debian unstable and in backports for 9 (Stretch).
+Cockpit is included in Debian unstable and in backports for 10 (Buster).
 
 1. For Debian 9 you have to enable the [backports repository](https://backports.debian.org):
```
Comment (martinpitt) [inline running.md:207]:
```suggestion
1. For Debian 10 you have to enable the [backports repository](https://backports.debian.org):
```

Another 9 :-)

**Review by martinpitt — CHANGES_REQUESTED** [review]

**Review by martinpitt — APPROVED** [review]
Danke!

**Pull request merged**